### ISSUE TITLE
Request function-set frequency manually

### ISSUE THREAD

**Issue by leon-uhs** [luispl77/433Screen-SignalHacker#3]
Can you add the function of manually setting frequency? Some places need 315Mhz, or other frequencies.

thank you

**Comment by luispl77** [comment]
Yes I will use the EEPROM to store all settings, so that they remain after the device is powered off. This means you will only need to change frequency once, in Replay/Store -> Receiver Config. Note that you need to get the right RFM69HW module for the right frequency band, there are [3 different versions](https://pt.aliexpress.com/item/33017904081.html?spm=a2g0o.productlist.main.5.67211f058mlRw7&algo_pvid=d2d488e0-b41d-4ffd-a371-56011c9deb81&algo_exp_id=d2d488e0-b41d-4ffd-a371-56011c9deb81-2&pdp_npi=3%40dis%21EUR%213.07%212.46%21%21%21%21%21%40211bf14716853442561715414d082f%2167142471256%21sea%21PT%21888150602&curPageLogUid=aUJgxROAhUK0) :  433/315 MHz, 868 MHz, 915 MHz . If you get a module for 433 MHz and try to use it in 868 MHz band, you will see a significant reduction in efficiency.

**Comment by leon-uhs** [comment]
Thank you very much. I'm making PCB. Looking forward to your Android Stm32+cc1101.
